### PR TITLE
[ru] Add RU localization for system-logs.md

### PR DESCRIPTION
**Pull request by kirkonru**
Files added and translated:

- [system-logs.md](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/cluster-administration/system-logs.md)

### CODE REVIEW

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+---
+reviewers:
```
Comment (sftim) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:2]:
```suggestion
```

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+## {{% heading "whatsnext" %}}
+
+* [Архитектура логирования в Kubernetes](/docs/concepts/cluster-administration/logging/)
+* [Структурированное логирование](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)
+* [Контекстное логирование](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging)
+* [Вывод флагов klog из эксплуатации](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
+* [Соглашения и правила для определения критичности логов](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)
```
Comment (sftim) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:186]: Optionally, mark these 4 links as being to English pages.

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+- `--log-dir`
+- `--log-file`
+- `--log-file-max-size`
+- `--logtostderr`
+- `--one-output`
+- `--skip-headers`
+- `--skip-log-headers`
+- `--stderrthreshold`
+
+Вывод всегда будет осуществляться в stderr независимо от его формата. Перенаправление вывода будет осуществляться компонентом, который вызывает компонент Kubernetes, например, POSIX-совместимой оболочкой или инструментом вроде systemd.
```
Comment (shurup) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:34]:
```suggestion
Вывод всегда будет записываться в stderr независимо от его формата. Перенаправление вывода должно осуществляться компонентом, который вызывает компонент Kubernetes, например, POSIX-совместимой командной оболочкой или инструментом вроде systemd.
```

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+В некоторых случаях, например, контейнер без дистрибутива или системная служба Windows, эти опции недоступны. В этом случае [`kube-log-runner`](https://github.com/kubernetes/kubernetes/blob/d2a8a81639fcff8d1221b900f66d28361a170654/staging/src/k8s.io/component-base/logs/kube-log-runner/README.md) можно использовать в качестве обертки вокруг компонента Kubernetes для перенаправления вывода. Его предварительно собранный исполняемый файл включен в несколько базовых образов Kubernetes под старым именем `/go-runner`. В архивах релизов серверов и узлов его можно найти как `kube-log-runner`.
```
Comment (shurup) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:36]: [This](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#server-binaries) is an example of the archives mentioned here. I suggest a bit different description for them:
```suggestion
Иногда эти опции недоступны — например, в случае контейнера без дистрибутива (distroless) или системной службы Windows. Тогда [`kube-log-runner`](https://github.com/kubernetes/kubernetes/blob/d2a8a81639fcff8d1221b900f66d28361a170654/staging/src/k8s.io/component-base/logs/kube-log-runner/README.md) можно использовать в качестве обертки вокруг компонента Kubernetes для перенаправления вывода. Его предварительно собранный исполняемый файл включен в некоторые базовые образы Kubernetes под старым именем `/go-runner`, а в актуальных бинарных релизах архивов с kubernetes-server и kubernetes-node он называется `kube-log-runner`.
```

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+which has a line break.
+```
+
+
+### Структурированное логирование
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+{{< warning >}}
+Переход на структурированное логирование — продолжающийся процесс. Не все сообщения структурированы в этой версии. При парсинге файлов логов необходимо также обрабатывать неструктурированные сообщения.
```
Comment (shurup) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:66]:
```suggestion
Переход на структурированное логирование — продолжающийся процесс. Не все сообщения структурированы в текущей версии Kubernetes. При парсинге файлов логов необходимо также обрабатывать неструктурированные сообщения.
```

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+<klog header> "<message>" <key1>="<value1>" <key2>="<value2>" ...
+```
+
+Пример:
+
+```ini
+I1025 00:15:15.525108       1 controller_utils.go:116] "Pod status updated" pod="kube-system/kubedns" status="ready"
+```
+
+Строки заключаются в кавычки. Другие значения форматируются с помощью [`%+v`](https://pkg.go.dev/fmt#hdr-Printing). В результате сообщение может продолжится на следующей строке в [зависимости от типа данных](https://github.com/kubernetes/kubernetes/issues/106428).
```
Comment (shurup) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:85]:
```suggestion
Строки заключаются в кавычки. Другие значения форматируются с помощью [`%+v`](https://pkg.go.dev/fmt#hdr-Printing). В результате сообщение может продолжиться на следующей строке в [зависимости от типа данных](https://github.com/kubernetes/kubernetes/issues/106428).
```

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+### Контекстное логирование
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+Контекстное логирование базируется на структурированном логировании. Речь идет в первую очередь о том, как разработчики используют лог-вызовы: код, основанный на этой концепции, более гибок и поддерживает дополнительные сценарии использования (см. [Contextual Logging KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging)).
+
+При использовании в компонентах дополнительных функций, таких как `WithValues` или `WithName`, записи лога содержат дополнительную информацию, которая передается в функции вызывающей стороной.
+
+В настоящее время за включение контекстного логирования отвечает переключатель функционала `StructuredLogging`. По умолчанию оно отключено. Соответствующая инфраструктура появилась в версии 1.24 без изменения компонентов. Команда [`component-base/logs/example`](https://github.com/kubernetes/kubernetes/blob/v1.24.0-beta.0/staging/src/k8s.io/component-base/logs/example/cmd/logger.go) показывает, как использовать новые лог-вызовы и как ведет себя компонент, поддерживающий контекстное логирование.
```
Comment (shurup) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:100]:
```suggestion
В настоящее время за включение контекстного логирования отвечает переключатель функционала `StructuredLogging`. По умолчанию оно отключено. Соответствующая инфраструктура появилась в версии 1.24 и она не потребовала изменений в компонентах. Команда [`component-base/logs/example`](https://github.com/kubernetes/kubernetes/blob/v1.24.0-beta.0/staging/src/k8s.io/component-base/logs/example/cmd/logger.go) показывает, как использовать новые лог-вызовы и как ведет себя компонент, поддерживающий контекстное логирование.
```

File: content/ru/docs/concepts/cluster-administration/system-logs.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+
+### Местоположение лога
+
+Существует два типа системных компонентов: те, которые работают в контейнере, и те, которые работают за пределами контейнера. Например:
+
+* Планировщик Kubernetes и kube-proxy работают в контейнере.
+* kubelet и {{<glossary_tooltip term_id="container-runtime" text="среда исполнения для контейнеров">}}
+  работают за пределами контейнеров.
+
+На машинах с systemd среда исполнения и kubelet пишут в journald. В противном случае ведется запись в файлы `.log` в директории `/var/log`. Системные компоненты внутри контейнеров всегда пишут в файлы `.log` в директории `/var/log`, обходя механизм логирования по умолчанию. как и логи контейнеров, логи системных компонентов в `/var/log` нуждаются в ротации. В кластерах Kubernetes, созданных с помощью скрипта `kube-up.sh`, ротация логов настраивается с помощью инструмента `logrotate`. `logrotate` ротирует логи ежедневно или при достижении ими размера в 100 МБ.
```
Comment (shurup) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:178]:
```suggestion
На машинах с systemd среда исполнения и kubelet пишут в journald. В противном случае ведется запись в файлы `.log` в директории `/var/log`. Системные компоненты внутри контейнеров всегда пишут в файлы `.log` в директории `/var/log`, обходя механизм логирования по умолчанию. Как и логи контейнеров, логи системных компонентов в `/var/log` нуждаются в ротации. В кластерах Kubernetes, созданных с использованием скрипта `kube-up.sh`, ротация логов настраивается с помощью инструмента `logrotate`. `logrotate` ротирует логи ежедневно или при достижении ими размера в 100 МБ.
```

**Review by shurup — COMMENTED** [review]
:+1: Just a few minor suggestions from me.

**Comment by shurup** [comment]
@kirkonru hi! Please, let's resolve the suggestions from our reviews to merge this PR. Otherwise, it might become outdated.

/remove-lifecycle rotten

File: content/ru/docs/concepts/cluster-administration/system-logs.md (hunk 2 above)
Comment (kirkonru) [inline content/ru/docs/concepts/cluster-administration/system-logs.md:186]:
```suggestion
* [Структурированное логирование (EN)](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)
* [Контекстное логирование (EN)](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging)
* [Вывод флагов klog из эксплуатации (EN)](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
* [Соглашения и правила для определения критичности логов (EN)](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)
```

Comment (kirkonru) [reply content/ru/docs/concepts/cluster-administration/system-logs.md:186]: Thanks for the suggestion!

**Comment by kirkonru** [comment]
@shurup please merge.

**Comment by shurup** [comment]
Great, thank you!
/lgtm
/approve

**Pull request merged**